### DISCUSSION TITLE
Sorting Songs inside Playlist

### DISCUSSION THREAD

**Discussion by MisterCHOUN** [pear-devs/pear-desktop#1044] [Ideas]
Hello,

I request a plugin where you can sort Songs inside a playlist, like :
- Sort by name (A->Z or Z->A)
- Sort by recent add
- Sort by Artist
- Sort by Album
- Custom Sort

Youtube allows you to sort your list of Playlist :

![Sorting list of playlist](https://user-images.githubusercontent.com/65115291/221404788-1d9c00a0-a4e7-4c91-8543-5126093b85d8.PNG)

But not what is INSIDE the Playlist...

![Cannot sort songs in playlist](https://user-images.githubusercontent.com/65115291/221405219-c6cbde9d-6ff4-4f84-90b7-80befae91769.PNG)

Google still has not implemented something so obvious when making a Music App...

Thanks in advance.

**Comment by Araxeus** [comment]
Honestly I dont know what the YouTube Music devs are smoking

**Reply by MisterCHOUN** [reply]
Yeah that's so dumb XD.

So I never touched javascript before, Do you think it will be easy to code a little sorting program ?

Like : 
- just ctrl+c and ctrl+v the actual code of sorting the list of playlists
- and modify / tweak with the list of songs

(Just adding [A->Z] & [Z->A] will be enough)

**Reply by Araxeus** [reply]
No I don't think it will be easy at all. unless you know of a way to modify the song order

**Reply by MisterCHOUN** [reply]
Haiya...

Welp, guess that's it. Thanks for answering me.

I'll wait and see if th-ch, yourself or any of the collaborator would like to do this this side request (in your free time of course)

Either way it's not a big deal, keep up the good work ! This modded app is very cool :D.
